docs/source/examples/mg-mode-part-i.ipynb

```diff
@@ -0,0 +1,926 @@
+    "mox43_xsdata.set_scatter_matrix(scatter_matrix)\n",
+    "mox43_xsdata.set_fission([7.62704E-03, 8.76898E-04, 5.69835E-03, 2.28872E-02,\n",
+    "                          1.07635E-02, 2.32757E-01, 2.48968E-01])\n",
+    "mox43_xsdata.set_nu_fission([2.175300E-02, 2.535103E-03, 1.626799E-02, 6.547410E-02,\n",
+    "                             3.072409E-02, 6.666510E-01, 7.139904E-01])\n",
+    "mox43_xsdata.set_chi([5.87910E-01, 4.11760E-01, 3.39060E-04, 1.17610E-07,\n",
+    "                      0.000000E-00, 0.000000E-00, 0.000000E-00])\n",
+    "\n",
+    "mox7_xsdata = openmc.XSdata('mox7', groups)\n",
+    "mox7_xsdata.order = 0\n",
+    "mox7_xsdata.set_total([1.813230E-01, 3.343680E-01, 4.937850E-01, 5.912160E-01,\n",
+    "                       4.741980E-01, 8.336010E-01, 8.536030E-01])\n",
+    "mox7_xsdata.set_absorption([9.06570E-03, 4.29670E-03, 3.28810E-02, 1.22030E-01,\n",
+    "                            1.82980E-01, 5.68460E-01, 5.85210E-01])\n",
+    "scatter_matrix = \\\n",
+    "    [[[1.304570E-01, 4.179200E-02, 8.510500E-06, 5.132900E-09, 0.000000E-00, 0.000000E-00, 0.000000E-00],\n",
+    "      [0.000000E-00, 3.284280E-01, 1.643600E-03, 2.201700E-09, 0.000000E-00, 0.000000E-00, 0.000000E-00],\n",
+    "      [0.000000E-00, 0.000000E-00, 4.583710E-01, 2.533100E-03, 0.000000E-00, 0.000000E-00, 0.000000E-00],\n",
+    "      [0.000000E-00, 0.000000E-00, 0.000000E-00, 4.637090E-01, 5.476600E-03, 0.000000E-00, 0.000000E-00],\n",
+    "      [0.000000E-00, 0.000000E-00, 0.000000E-00, 1.761900E-04, 2.823130E-01, 8.728900E-03, 9.001600E-09],\n",
+    "      [0.000000E-00, 0.000000E-00, 0.000000E-00, 0.000000E-00, 2.276000E-03, 2.497510E-01, 1.311400E-02],\n",
+    "      [0.000000E-00, 0.000000E-00, 0.000000E-00, 0.000000E-00, 0.000000E-00, 8.864500E-03, 2.595290E-01]]]\n",
+    "scatter_matrix = np.array(scatter_matrix)\n",
+    "scatter_matrix = np.rollaxis(scatter_matrix, 0, 3)\n",
+    "mox7_xsdata.set_scatter_matrix(scatter_matrix)\n",
+    "mox7_xsdata.set_fission([8.25446E-03, 1.32565E-03, 8.42156E-03, 3.28730E-02,\n",
+    "                         1.59636E-02, 3.23794E-01, 3.62803E-01])\n",
+    "mox7_xsdata.set_nu_fission([2.381395E-02, 3.858689E-03, 2.413400E-02, 9.436622E-02,\n",
+    "                            4.576988E-02, 9.281814E-01, 1.043200E+00])\n",
+    "mox7_xsdata.set_chi([5.87910E-01, 4.11760E-01, 3.39060E-04, 1.17610E-07,\n",
+    "                     0.000000E-00, 0.000000E-00, 0.000000E-00])\n",
+    "\n",
+    "mox87_xsdata = openmc.XSdata('mox87', groups)\n",
+    "mox87_xsdata.order = 0\n",
+    "mox87_xsdata.set_total([1.830450E-01, 3.367050E-01, 5.005070E-01, 6.061740E-01,\n",
+    "                        5.027540E-01, 9.210280E-01, 9.552310E-01])\n",
+    "mox87_xsdata.set_absorption([9.48620E-03, 4.65560E-03, 3.62400E-02, 1.32720E-01,\n",
+    "                             2.08400E-01, 6.58700E-01, 6.90170E-01])\n",
+    "scatter_matrix = \\\n",
```

Comment:
Better idea, with just the uo2_data, I now show how one would write a library, but then I end up saying 'but im just going to use a library I already made, called `c5g7.h5` and skip over those other data sets.

- [x]